From 8858b8eacd1385f99796ead78b3a7157c5e93121 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Sun, 24 Jun 2012 14:57:14 +0200
Subject: [PATCH 1/6] build: remove unnecessary pkgconfig->config.status
 dependency

The flags obtained from pkgconfig are in essence preprocessor ones.

Signed-off-by: Jan Engelhardt <jengelh@inai.de>
---
 Make_global.am |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Make_global.am b/Make_global.am
index b7dab7b..a4e9bd9 100644
--- a/Make_global.am
+++ b/Make_global.am
@@ -1,2 +1,2 @@
-AM_CPPFLAGS = -I${top_srcdir}/include
-AM_CFLAGS = -Wall ${LIBNFNETLINK_CFLAGS}
+AM_CPPFLAGS = -I${top_srcdir}/include ${LIBNFNETLINK_CFLAGS}
+AM_CFLAGS = -Wall
-- 
1.7.10.4

